guides/common/modules/proc_increasing-the-logging-level-for-pulp.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-pulp"]
+= Increasing the logging level for Pulp
+
+By default, Pulp logs to `syslog`.
+You can view the log in `/var/log/messages` or with `journalctl`.
```

Comment:
Technically it logs to journald. By default journald is configured to forward to syslog, but that's optional and users can choose to disable that. Then it won't be in `/var/log/messages`. I'd discourage that and encourage using `journalctl`.

Reply:
This is a bit unclear to me. Do you have a specific suggestion in mind?